### PR TITLE
[VRE 👍 ] 29552 Send comm preferences in VRE summary

### PR DESCRIPTION
**Pull request by RudyOnRails**
## Description of change

This PR does a few things:
2. adjusts mailer to include two new columns, with test
3. removes brittle fixture from mailer spec and replaces with standard `include` matcher
4. add preview mailer

## Original issue(s)
https://github.com/department-of-veterans-affairs/va.gov-team/issues/29552

### CODE REVIEW

File: lib/boolean_helper.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+module BooleanHelper
+  def boolean_to_yesno(boo)
+    boo ? 'Yes' : 'No'
+  end
```
Comment (ericboehs) [inline lib/boolean_helper.rb:6]: This seems a bit overkill for just two variables.

```
boolean_to_yesno(boo)
```
is longer than 
```
boo ? 'Yes' : 'No'
```
and adds a layer of abstraction so you have to go look up what it does.

I understand avoiding writing logic in a view but conditionals of this simplicity seem okay.

Could you explain more why you added this help? I think I'm missing something.

Comment (RudyOnRails) [reply lib/boolean_helper.rb:6]: My unit test on the helper did catch a failure (I expected "Yes" but was actually returning "yes" at first). Wouldn't it be more work to write a view spec? Would I have to revert to using a fixture of markup?

I am totally open to using your suggestion, but am curious about the testing of it.

Comment (RudyOnRails) [reply lib/boolean_helper.rb:6]: Good morning sir. If you get a second can you share your thoughts on this ☝️.

Comment (RudyOnRails) [reply lib/boolean_helper.rb]: Ok this has been resolved. Helper and spec removed. Logic moved to email view.

File: app/mailers/ch31_submissions_report_mailer.rb [hunk 2]
```diff
@@ -3,6 +3,8 @@
 require 'reports/uploader'
 
 class Ch31SubmissionsReportMailer < ApplicationMailer
+  include BooleanHelper
```
Comment (ericboehs) [inline app/mailers/ch31_submissions_report_mailer.rb:6]: This doesn't need to be here anymore, right?

Comment (RudyOnRails) [reply app/mailers/ch31_submissions_report_mailer.rb]: Yeah, CI just caught it, and it's gone.

**Review by ericboehs — APPROVED** [review]

**Pull request merged**